### PR TITLE
Implementing Date Flag

### PR DESCRIPTION
**Pull request by alexwilson**
Implementing a flag based on date, to set a concrete date for when features are enabled and sunset.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,6 +177,12 @@ device:
   name: regex # give regex for each valid device
 ```
 
+``` yaml
+date:
+  start_date: "2016-09-01" # (optional) start date, accepts DateTime constructor values (php.net/datetime)
```
Comment (DZunke) [inline README.md]: May you could note the default value?

File: Toggle/Conditions/Date.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace DZunke\FeatureFlagsBundle\Toggle\Conditions;
+
+use DateTime;
+
+class Date extends AbstractCondition implements ConditionInterface
+{
+    /**
+     * @var date
```
Comment (DZunke) [inline Toggle/Conditions/Date.php]: I think the correct type would be `string` :smile:

**Comment by DZunke** [comment]
Hey! Thanks for your contribution. It seems to work great but there are two small style fixes that would be nice to add. If done i will merge and create a new release.

**Comment by alexwilson** [comment]
Done!

**Comment by DZunke** [comment]
:+1: Thanks!

**Pull request merged**